scania-digital-design-system/sdds
# fix(toast): renamed role to toastRole

**Describe pull-request**  
Renamed role prop to toastRole. Stencil 3 does not accept role as a prop name, since it clashes with native role attribute.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below
2. Check in Components -> Toast -> Web Components
3. Check in readme.

**Suggested test steps**
